packages/ibm-products/src/components/Card/CardFooter.js

```diff
@@ -72,12 +75,23 @@ export let CardFooter = ({
         </Button>
       )}
       {hasActions && <div className={`${blockClass}__actions`}>{actions}</div>}
+      {FooterActionIcon && (
+        <div className={`${blockClass}__action-icon`}>
+          {disabled ? (
+            <Error size={16}></Error>
```

Comment:
you should be able to use the shorthand notation for consistency sake `<Error />` `<FooterActionIcon />`